### PR TITLE
Include fan-out operators in the operator index

### PR DESCRIPTION
**Pull request by raboof**
Also helps resolve https://github.com/akka/akka.github.com/issues/606

### CODE REVIEW

File: akka-docs/src/main/paradox/stream/operators/index.md [hunk 1]
```diff
@@ -257,6 +257,22 @@ the inputs in different ways.
 |Source/Flow|<a name="zipwith"></a>@ref[zipWith](Source-or-Flow/zipWith.md)|Combines elements from multiple sources through a `combine` function and passes the returned value downstream.|
 |Source/Flow|<a name="zipwithindex"></a>@ref[zipWithIndex](Source-or-Flow/zipWithIndex.md)|Zips elements of current flow with its indices.|
 
+## Fan-out operators
+
+These have one input and multiple outputs. They might route the elements between different outputs, or emit elements on
+multiple outputs at the same time.
+
+There is currently no 'fluent' API available for the fan-out operators, to use them you will have to use the @ref[Graph DSL](../stream-graphs.md#constructing-graphs).
```
Comment (helena) [inline akka-docs/src/main/paradox/stream/operators/index.md]:
```suggestion
A 'fluent' API for the fan-out operators is not yet available, until then use the @ref[Graph DSL](../stream-graphs.md#constructing-graphs).
```

**Review by helena — APPROVED** [review]
One minor doc suggestion, otherwise LGTM

File: akka-docs/src/main/categories/fan-out-operators.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 These have one input and multiple outputs. They might route the elements between different outputs, or emit elements on
-multiple outputs at the same time.
\ No newline at end of file
+multiple outputs at the same time.
+
+There is currently no 'fluent' API available for the fan-out operators, to use them you will have to use the @ref[Graph DSL](../stream-graphs.md#constructing-graphs).
```
Comment (johanandren) [inline akka-docs/src/main/categories/fan-out-operators.md]: Actually, at few of them, or special cases of them, do have 'fluent' API: `alsoTo`, `divertTo` and `wireTap`.

Not sure if that is worth mentioning.

**Review by johanandren — APPROVED** [review]

File: akka-docs/src/main/categories/fan-out-operators.md (hunk 2 above)
Comment (raboof) [reply akka-docs/src/main/categories/fan-out-operators.md]: These are currently filed under 'Simple operators'. I agree 'Fan-out operators' is more specific and thus better, I'll move them

**Comment by johanandren** [comment]
Should it be backported @raboof ?

**Comment by raboof** [comment]
I don't think it needs to

**Pull request merged**